multisense_ros/src/camera.cpp

```diff
@@ -348,7 +348,8 @@ Camera::Camera(const std::string& node_name,
     // S27/S30 cameras have a 3rd aux color camera and no left color camera
 
     has_aux_camera_ = system::DeviceInfo::HARDWARE_REV_MULTISENSE_C6S2_S27 == device_info_.hardwareRevision ||
-                      system::DeviceInfo::HARDWARE_REV_MULTISENSE_S30 == device_info_.hardwareRevision;
+                      system::DeviceInfo::HARDWARE_REV_MULTISENSE_S30 == device_info_.hardwareRevision ||
+					  system::DeviceInfo::HARDWARE_REV_MULTISENSE_KS21i == device_info_.hardwareRevision;
```

Comment:
The spacing seems off here

Reply:
Good catch, there were some tabs hiding there that happened to look ok in my editor, fixed.